railties/lib/rails/application/configuration.rb

```diff
@@ -52,6 +52,13 @@ def initialize(*)
         @x                             = Custom.new
       end
 
+      def static_cache_control=(value)
+        ActiveSupport::Deprecation.warn("static_cache_control is deprecated and will be removed in Rails 5.1. " \
+                                        "Please use `config.public_file_server.headers['Cache-Control'] = #{value} instead.")
```

Comment:
Would be good to have this as-
```ruby
config.public_file_server.headers= {'Cache-Control' => #{value}}
```
This would suggest the options being passed to headers have been expanded.